packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1399,6 +1400,30 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   }
 
+  /**
+   * post Event, provide the ability to send Call Analyzer for the request of third-party
+   * @public
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  // eslint-disable-next-line valid-jsdoc
+  public postEvent(eventName) {
```

Comment:
the name postEvent() is too generic, the method name should be postMetric(). BTW, I don't see the method being called from anywhere, so is it really needed? Also we should specify the type for the eventName argument and the docs are missing for it

Reply:
ah, sorry I've realised now that this method is meant to be called by the app, so it's ok that it's not called from within SDK, but that makes it even more important for the docs and types to be fully populated and correct.

Reply:
Yes, This is just an exposed port, which is convenient for the app to pass some Call analyzer data. We know that some post Mertris behaviors must be performed on many page UI terminals (login page, interstitial page etc.) to make basic data for the JMF & JMS report analysis.